sockeye/data_io.py

```diff
@@ -20,13 +20,15 @@
 import os
 import pickle
 import random
+import time
 from abc import ABC, abstractmethod
 from collections import OrderedDict
 from contextlib import ExitStack
 from typing import Any, cast, Dict, Iterator, Iterable, List, Optional, Sequence, Sized, Tuple
 
 import mxnet as mx
 import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
this does not seem to be used and should be removed. matplotib is just an optional dependency for Sockeye.

Reply:
Oh good catch. I am going to remove it.